src/marqo/s2_inference/s2_inference.py

```diff
@@ -13,12 +13,12 @@
 available_models = dict()
 MODEL_PROPERTIES = load_model_properties()
 
-def vectorise(model_name: str, content: Union[str, List[str]], device: str = get_default_device(), 
+def vectorise(model: Union[str, dict], content: Union[str, List[str]], device: str = get_default_device(),
```

Comment:
this is incorrectly named - either use the old name (`model_name` or also modify all the instances that call vectorise